cpp/dolfinx/fem/Form.h

```diff
@@ -49,30 +49,39 @@ struct integral_data
   /// @param id Domain ID.
   /// @param kernel Integration kernel.
   /// @param entities Entities to integrate over.
-  template <typename K, typename V>
+  /// @param enabled_coefficients Coefficients in form that are enabled in this
+  /// integral
+  template <typename K, typename V, typename Q>
```

Comment:
What is `Q`? For the other templated args we use concepts to constrain what they must be able to do.

Reply:
I am not sure what you are after here. We need the input to be convertible to a `std::vector<std::int8_t>`. What would you do differently?

Reply:
Did I miss the `std::is_convertible_v<std::remove_cvref_t<Q>, ...` on my first pass?